pallets/system/src/extensions/check_genesis.rs

```diff
@@ -46,6 +46,7 @@ impl<T: Config + Send + Sync> sp_std::fmt::Debug for CheckGenesis<T> {
 
 impl<T: Config + Send + Sync> CheckGenesis<T> {
 	/// Creates new `SignedExtension` to check genesis hash.
+	#[allow(clippy::new_without_default)]
```

Comment:
I think is better to add `default` here, instead of mute clippy.

Reply:
Fixed accordingly